src/elm/3_moduler_elm_reactor/3_moduler_elm_reactor.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Elm reactor og moduler"
+level: 2
+language: nb-NO
+author: Alexander Perry
+---
+
+# Introduksjon {.intro}
+
+Denne oppgaven innfører moduler, som lar oss bruke kode som andre har skrevet.
+Når vi forstår moduler, er vi klare for `elm reactor`, som viser oss en
+webapplikasjon vi har skrevet i vår egen fil.
```

Comment:
Elm-plattformen kommer med fire programmer:

- `elm repl` starter en read-eval-print-loop, på samme måte som når man skriver `python` i terminalen
- `elm reactor` overvåker en mappe med elm-programmer så vi kan oppdatere nettleseren uten å måtte rekompilere
- `elm make` lager HTML og/eller Javascript-filer fra Elm-filer
- `elm package` kan installere og oppgradere pakker vi bygger på.

Tanken med 
> som viser oss en webapplikasjon vi har skrevet i vår egen fil
er at den kan fylle samme rolle som **Try Elm** for lokale filer.

Synes dette er litt vanskelig å forklare skriftlig (blir mye å lese for barna), så jeg har tatt det muntlig.

Jeg har skrevet `elm reactor` som kode for å signalisere at det er eksakt dette vi skriver i en kommandolinje for å starte programmet.